lib/commands/general.js

```diff
@@ -45,9 +45,9 @@ commands.getStrings = async function (language) {
   return this.apkStrings[language];
 };
 
-commands.getWindowSize = async function () {
+commands.getWindowSize = _.memoize(async function () {
```

Comment:
@imurchie I assume this should be rather memoized per instance, but not per class